modules/porous_flow/doc/content/modules/porous_flow/natural_convection.md

```diff
@@ -0,0 +1,94 @@
+
+
+!table id=datatable caption=Parameters of the problem [!citep](Oldenburg1995OnTD).
+| Symbol | Quantity | Value | Units |
+| --- | --- | --- | --- |
+| $\phi$ | Porosity | $0.1$ | $-$ |
+| $k$ | Permeability | $1.21*10^{-10}$  | $m^2$ |
+| $\mu$ | Viscosity ($T=20^oC$) | $1*10^{-3}$  | $J.Kg^{-1}.s^{-1}.m^{-1}$ |
+| $K$ | Thermal conductivity | $1.49$  | $J.Kg^{-1}.^oC^{-1}$ |
+| $C$ | Heat capacity of rock | $0$ | $m^2$ ||
+| $g$ | Gravity | $9.81$  | $m.s^{2}$ |
+| $T_0$ | Initial temperature | $12$  | $^oC$ |
+
+!media media/porous_flow/Convec_schematic.png style=width:100%;margin-left:10px; caption=The experiment schematic [!citep](Oldenburg1995OnTD). id=schematicFig
+
+
+## Input file setup
+
+This section describes the input file syntax.
+
+### Mesh
+
+The first step is to set up the mesh. In this problem, a rectangular tank with a width of 300 m and a height of 150 m is simulated. For convenience, instead of the YZ coordinate as shown in [schematicFig], the simulation was set up in XY coordinate with the positive Y direction pointing toward the top of the tank. In this code block, the numbers of nodes in the X and Y directions are specified along with the dimension of the tank. A notable feature is the declaration of the boundary "Heater". This is due to the fact that the heat element at the bottom of the tank just located between 0< x < 150 m. This enables simple boundary condition implementation at later stages.
+!listing modules/porous_flow/examples/natural_convection/natural_convection.i block=Mesh
+
+### Fluid properties
+
+After the mesh has been specified, we need to provide the fluid properties that are used for this simulation. For our case, it will be just water and can be easily implemented as follows:
+
+!listing modules/porous_flow/examples/natural_convection/natural_convection.i block=FluidProperties
+
+### Variables declaration
+
+Then, we need to declare what type of unknown we are solving. This can be done in the variable code block. Since we are focusing on the natural convection of flow in a porous medium, temperature (T) is a desired variable. Porepressure is also used for the fluid motion.
+
+!listing modules/porous_flow/examples/natural_convection/natural_convection.i block=Variables
+
+### Action declaration
+
+This section is to tell the solver what problem we need to solve. To do so, an action is declared. In MOOSE, an action will help users automatically add all the kernels and objects required for this problem. For this problem, the action "PorousFlowFullySaturated" with "Thermohydro" coupling is activated to account for heat convection. To have a better understanding, users are recommended to visit [this page](https://mooseframework.inl.gov/source/actions/PorousFlowFullySaturated.html).
```

Comment:
```suggestion
 For this problem, we use the [`PorousFlowFullySaturated`](https://mooseframework.inl.gov/source/actions/PorousFlowFullySaturated.html) to automatically add all of the physics kernels and a number of the necessary material properties. As this is a TH problem, we set `coupling_type = Thermohydro`.
```

Reply:
Also, the link to the page is just a relative link 
`[`PorousFlowFullySaturated`](PorousFlowFullySaturated.md)`

Reply:
Same with all the other links - you can link to the md file like above and it will make sure it is correct when the website is built.